### PR TITLE
Add hospital admissions prop signal to CPR

### PR DESCRIPTION
**Pull request by nmdefries**
### Description
Need prop signal to support dashboard. Add prop signal creation function and tests. Introduce proportion flag as third value in dfs key tuple.

### Changelog
- `constants.py`
- `pull.py`
- `run.py`
- `test_pull.py`
- Local and ansible params templates.

### CODE REVIEW

File: dsew_community_profile/delphi_dsew_community_profile/constants.py [hunk 1]
```diff
@@ -50,22 +50,30 @@ class Transform:
...
+        "api_prop_name": "confirmed_admissions_covid_1d_prop_7dav"
     }
 }
 
 COUNTS_7D_SIGNALS = {key for key, value in SIGNALS.items() if not value["is_rate"]}
 
-def make_signal_name(key):
+def make_signal_name(key, is_prop=False):
     """Convert a signal key to the corresponding signal name for the API."""
+    if is_prop:
```
Comment (krivard) [inline dsew_community_profile/delphi_dsew_community_profile/constants.py:73]:
```suggestion
def make_signal_name(key, is_prop=False):
    """Convert a signal key to the corresponding signal name for the API.
    
    Note, this function gets called twice with the same `key` for signals that support 
    population-proportion ("prop") variants.
    """
    if is_prop:
```

File: dsew_community_profile/delphi_dsew_community_profile/constants.py [hunk 2]
```diff
@@ -50,22 +50,30 @@ class Transform:
...
+        return SIGNALS[key]["api_prop_name"]
     return SIGNALS[key]["api_name"]
 
-NEWLINE="\n"
+NEWLINE = "\n"
+NOT_PROP = "not prop"
+IS_PROP = "prop"
```
Comment (krivard) [inline dsew_community_profile/delphi_dsew_community_profile/constants.py:79]: any reason you're not just handling these as straight booleans?

**Review by krivard — APPROVED** [review]
Broadly good, excellent tests, small efficiency nits (optional) & additional clarifying documentation (not)

File: dsew_community_profile/delphi_dsew_community_profile/constants.py (hunk 2 above)
Comment (nmdefries) [reply dsew_community_profile/delphi_dsew_community_profile/constants.py:79]: Ah, good point. I originally didn't have the constants defined so having strings was clearer.

**Review by krivard — APPROVED** [review]
🎉

**Pull request merged**